addons/nuxeo-retention/nuxeo-retention-core/src/main/java/org/nuxeo/retention/listeners/RetentionBusinessEventListener.java

```diff
@@ -78,4 +84,17 @@ public void handleEvent(Event event) {
         }
     }
 
+    protected boolean isExpression(String expression) {
+        UserManager userManager = Framework.getService(UserManager.class);
+        try (CloseableCoreSession session = CoreInstance.openCoreSession(null)) {
```

Comment:
No, I meant `openCoreSessionSystem`, not `openCoreSession`. You want the system user, right?

Reply:
yes, sorry